networking/p2p/build.gradle

```diff
@@ -7,7 +7,9 @@ jar {
 }
 
 dependencies {
-
+   implementation project(':util')
```

Comment:
Is there an extra space here or is it just rendering wrong?

Reply:
Sorry, confusing terminology. Newline was removed but it looks like there's 3 spaces indenting Line 19, whereas there's 2 on each line under it.

Reply:
you aren't human

Reply:
LOL